frontend/src/features/splReports/components/ElipsisControls.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { FaEllipsisH } from 'react-icons/fa';
+import { Dropdown, DropdownButton } from 'react-bootstrap';
+import { IReport } from '../interfaces';
+import './ElipsisControls.scss';
+
+interface IElipsisControlsProps {
+  /** The underlying report that this control is mapped to. */
+  report: IReport;
+  /** function to invoke when the open dropdown item is clicked */
+  onOpen: (report: IReport) => void;
+  /** function to invoke when the Mark as Final dropdown item is clicked */
+  onFinal: (report: IReport) => void;
+  /** function to invoke when the open dropdown item is clicked */
+  onDelete: (report: IReport) => void;
+}
+
+/**
+ * Dropdown controls spawned from an elipsis button. uses open, final, and delete functions.
+ */
+const ElipsisControls: React.FunctionComponent<IElipsisControlsProps> = ({
```

Comment:
Don't change, but looks like an opportunity for a customizable component.  Essentially a button that displays a menu.